File: TextAnalysisFront/src/app/repository/baseInterfaces/ISingleWordsRepository.ts
export interface ISingleWordsRepository {
    GetAllWords(): void;
    PostWord(word:string, mongoId:string): void;
    PutWord(word_to_replace:string, word:string, mongoId:string): void;
    DeleteWord(wordToRemove:string): void;
    DeleteCollection(): void;
}

File: TextAnalysisNetServer/Repository/BaseInterfaces/ISingleWordsRepository.cs
using System.Collections.Generic;

namespace TextAnalysis
{
	public interface ISingleWordsRepository
	{
		List<string> GetAllWords();
		string PostWord(string word);
		string PutWord(string word, string connectionWord);
		int DeleteWord(string word);
		int DeleteCollection();
		bool IfWordExists(string word);
	}
}
